svg/content-type-with-parameters-ref.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>SVG data is recognized even when the 'Content-Type' headers includes parameters</title>
```

Comment:
Is it really SVG? This is not related to any SVG features, but to HTTP in fact.
Should it be in `html/rendering/replaced-elements`
or in `html/semantics/embedded-content/the-img-element`